myriad-scheduler/src/main/java/com/ebay/myriad/scheduler/fgs/OfferUtils.java

```diff
@@ -0,0 +1,35 @@
+package com.ebay.myriad.scheduler.fgs;
+
+import java.util.Collection;
+import org.apache.hadoop.yarn.api.records.Resource;
+import org.apache.mesos.Protos;
+import org.apache.mesos.Protos.Offer;
+
+/**
+ * Utility class that provides useful methods that deal with Mesos offers.
+ */
+public class OfferUtils {
+
+  /**
+   * Transforms a collection of mesos offers into {@link Resource}.
+   *
+   * @param offers collection of mesos offers
+   * @return a single resource object equivalent to the cumulative sum of mesos offers
+   */
+  public static Resource getYarnResourcesFromMesosOffers(Collection<Offer> offers) {
+    double cpus = 0.0;
+    double mem = 0.0;
+
```

Comment:
I now see your point. Any idea how other mesos frameworks handle this? Sounds like every framework needs this ratio for CPUs. 

@adam-mesos Are there any guidelines for framework developers around how to interpret Mesos offered CPU resources?